android/sdk/src/main/java/com/taobao/weex/utils/WXViewUtils.java

```diff
@@ -280,6 +281,42 @@ public static float getRealPxByWidth(float pxValue,int customViewport) {
       return realPx > 0.005 && realPx < 1 ? 1 : (float) Math.rint(realPx);
     }
   }
+    public static float getRealPxByWidth(Object value, float df, int customViewport) {
```

Comment:
Please mark the following method as `@Deprecated` and `Use {@link #getRealPxByWidth(Object, float, int)}` instead.
* `getRealPxByWidth(float pxValue)`
* `getRealPxByWidth(float pxValue,int customViewport)`